### PR TITLE
Add Motion Blinds Integration

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add the Motion Blinds Integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/42989
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2013
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/motion_blinds.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{% configuration %}
+host:
+  description: IP address of the Motion Bridge, e.g., 192.168.1.100.
+  required: true
+  type: string
+API Key:
+  description: The API Key of your Motion Bridge, see [Retrieving the API Key](#retrieving-the-api-key).
+  required: true
+  type: string
+{% endconfiguration %}
```
Comment (frenck) [inline source/_integrations/motion_blinds.markdown]: This is used for YAML configuration entries.

File: source/_integrations/motion_blinds.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - Cover
+ha_iot_class: Local Polling
+ha_release: 0.119.0
+ha_domain: motion_blinds
+ha_codeowners:
+  - '@starkillerOG'
+ha_config_flow: true
+---
+
+The `motion_blinds` platform allows you to control [Motion Blinds](https://motion-blinds.com) from [Coulisse B.V.](https://coulisse.com/products/motion) within Home Assistant.
```
Comment (frenck) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
The integration allows you to control [Motion Blinds](https://motion-blinds.com) from [Coulisse B.V.](https://coulisse.com/products/motion).
```

File: source/_integrations/motion_blinds.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  type: string
+API Key:
+  description: The API Key of your Motion Bridge, see [Retrieving the API Key](#retrieving-the-api-key).
+  required: true
+  type: string
+{% endconfiguration %}
+
+Note that a static IP address is required in order for this integration to keep working properly.
+
+###Retrieving the API Key
```
Comment (frenck) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
## Retrieving the API Key

```

File: source/_integrations/motion_blinds.markdown [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+Retrieving Key
+
+The Motion Blinds API uses a 16 character key that can be retrieved from the official "Motion Blinds" app for [IOS](https://apps.apple.com/us/app/motion-blinds/id1437234324) or [Android](https://play.google.com/store/apps/details?id=com.coulisse.motion).
+Open the app, click the 3 dots in the top right corner, go to "settings", go to "Motion APP About", Please quickly tap this "Motion APP About" page 5 times, a popup will apear that gives you the key.
+Please note that "-" characters need to be included in the key when providing it to Home Assistant. The key needs to be simular to "12ab345c-d67e-8f"
+
+![alt text](https://raw.githubusercontent.com/starkillerOG/motion-blinds/main/pictures/Motion_App__get_key_1.jpg)
+![alt text](https://raw.githubusercontent.com/starkillerOG/motion-blinds/main/pictures/Motion_App__get_key_2.jpg)
+
+[Motion Blinds]: /integrations/motion_blinds
```
Comment (frenck) [inline source/_integrations/motion_blinds.markdown]:
```suggestion

The Motion Blinds API uses a 16 character key that can be retrieved from the official "Motion Blinds" app for [IOS](https://apps.apple.com/us/app/motion-blinds/id1437234324) or [Android](https://play.google.com/store/apps/details?id=com.coulisse.motion).

Open the app, click the 3 dots in the top right corner, go to "settings", go to "Motion APP About", Please quickly tap this "Motion APP About" page 5 times, a popup will appear that gives you the key.

Please note that "-" characters need to be included in the key when providing it to Home Assistant. The key needs to be similar to `12ab345c-d67e-8f`

![alt text](https://raw.githubusercontent.com/starkillerOG/motion-blinds/main/pictures/Motion_App__get_key_1.jpg)
![alt text](https://raw.githubusercontent.com/starkillerOG/motion-blinds/main/pictures/Motion_App__get_key_2.jpg)

[Motion Blinds]: /integrations/motion_blinds

```

File: source/_integrations/motion_blinds.markdown [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+###Retrieving the API Key
+Retrieving Key
+
+The Motion Blinds API uses a 16 character key that can be retrieved from the official "Motion Blinds" app for [IOS](https://apps.apple.com/us/app/motion-blinds/id1437234324) or [Android](https://play.google.com/store/apps/details?id=com.coulisse.motion).
+Open the app, click the 3 dots in the top right corner, go to "settings", go to "Motion APP About", Please quickly tap this "Motion APP About" page 5 times, a popup will apear that gives you the key.
+Please note that "-" characters need to be included in the key when providing it to Home Assistant. The key needs to be simular to "12ab345c-d67e-8f"
+
+![alt text](https://raw.githubusercontent.com/starkillerOG/motion-blinds/main/pictures/Motion_App__get_key_1.jpg)
+![alt text](https://raw.githubusercontent.com/starkillerOG/motion-blinds/main/pictures/Motion_App__get_key_2.jpg)
```
Comment (frenck) [inline source/_integrations/motion_blinds.markdown]: These images need to be hosted from this repository.

File: source/_integrations/motion_blinds.markdown (hunk 4 above)
Comment (frenck) [inline source/_integrations/motion_blinds.markdown]: This link reference seems to be unused?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/motion_blinds.markdown (hunk 4 above)
Comment (starkillerOG) [reply source/_integrations/motion_blinds.markdown]: Sorry I will remove it, I saw that with another platform and thought that it was nessesary.

File: source/_integrations/motion_blinds.markdown (hunk 5 above)
Comment (starkillerOG) [reply source/_integrations/motion_blinds.markdown]: In which folder can I place the images?

Comment (starkillerOG) [reply source/_integrations/motion_blinds.markdown]: I placed them in source/images/integrations/motion_blinds/, I hope that is correct?

File: source/_integrations/motion_blinds.markdown (hunk 1 above)
Comment (starkillerOG) [reply source/_integrations/motion_blinds.markdown]: I removed this and rephrased the important information.

**Comment by starkillerOG** [comment]
@frenck could you merge this since the parent has been merged?
(I already made the changes you requested)

**Comment by frenck** [comment]
The parent PR has been merged 22 minutes ago... please have some patience, thanks! 👍

**Review by frenck — APPROVED** [review]
Thanks! 👍

**Pull request merged**